components/centraldashboard/app/server.ts

```diff
@@ -46,14 +47,20 @@ async function main() {
   app.use(express.json());
   app.use(express.static(frontEnd));
   app.use(attachUser(USERID_HEADER, USERID_PREFIX));
-  app.use(
-      '/api', new Api(k8sService, workgroupApi, metricsService).routes());
+  app.get('/debug', (req: Request, res: Response) => {
+    res.json({
+      user: req.user,
+      profilesServiceUrl,
+      codeEnvironment,
+    });
```

Comment:
should we also add the inferred Kubeflow version in this reponse?

Reply:
/api/env-info request can be used to query the version of Kubeflow as parsed by the CentralDashboard (to be added to a test)